src/components/utils/popover/components/popover-item/popover-item-default/popover-item-default.const.ts

```diff
@@ -23,4 +23,5 @@ export const css = {
   iconTool: className('icon', 'tool'),
   iconChevronRight: className('icon', 'chevron-right'),
   wobbleAnimation: bem('wobble')(),
+  line: className('line'),
```

Comment:
"line" is not used in "default" item. 

p.s. why not "delimiter"?

Reply:
fixed